Cargo.toml

```diff
@@ -33,6 +33,7 @@ async-stream = "0.3.0"
 async-trait = "0.1.48"
 fnv = "1.0.7"
 futures-util = { version = "0.3.0", default-features = false, features = ["io", "sink"] }
+hashbrown = { version="0.11.2", optional = true }
```

Comment:
Move this line after `# Feature optional dependencies`.

Reply:
Also fixed rustfmt error (always forget to enable in my IDE for new project =) ).